BangazonAPI/Models/TrainingProgram.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class TrainingProgram
+    {
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+
+        public int StartDate { get; set; }
+
+        public int EndDate { get; set; }
+    }
```

Comment:
Employees who signed up for a training program should be included in the response. Needs all the columns on the ERD.

Reply:
I added a list of employees to this class in the new updated code :)